src/query/ast/tests/it/testdata/statement-error.txt

```diff
@@ -515,6 +515,18 @@ error:
                 [ copyOptions ]`
 
 
+---------- Input ----------
```

Comment:
Don't understand the error hint and what I will do 😢

Reply:
you are right
let me rethink